manager/manager.go

```diff
@@ -119,12 +120,21 @@ type manager struct {
 	quitChannels           []chan error
 	cadvisorContainer      string
 	dockerContainersRegexp *regexp.Regexp
+	loadReader             *cpuload.CpuLoadReader
 }
 
 // Start the container manager.
 func (self *manager) Start() error {
+	// Create cpu load reader.
+	cpuLoadReader, err := cpuload.New()
+	if err != nil {
+		glog.Infof("could not initialize cpu load reader: %s", err)
```

Comment:
This is going to show up as a warning in all namespaced containers. We can update it to warning once we have the sched_debug backend.